packages/benchmark/src/n8nApiClient/workflowsApiClient.ts

```diff
@@ -0,0 +1,50 @@
+import { Workflow } from '@/n8nApiClient/n8nApiClient.types';
+import { AuthenticatedN8nApiClient } from './authenticatedN8nApiClient';
+
+export class WorkflowApiClient {
+	private readonly apiEndpoint = '/workflows';
+
+	constructor(private readonly apiClient: AuthenticatedN8nApiClient) {}
+
+	async getAllWorkflows(): Promise<Workflow[]> {
+		const response = await this.apiClient.get<{ count: number; data: Workflow[] }>(
+			this.apiEndpoint,
+		);
+
+		return response.data.data;
+	}
+
+	async createWorkflow(workflow: unknown): Promise<Workflow> {
+		const response = await this.apiClient.post<{ data: Workflow }>(this.apiEndpoint, workflow);
+
+		return response.data.data;
+	}
+
+	async activateWorkflow(workflow: Workflow): Promise<Workflow> {
+		const response = await this.apiClient.patch<{ data: Workflow }>(
+			`${this.apiEndpoint}/${workflow.id}`,
```

Comment:
Too dry? We're unlikely to change `/workflows`, might as well keep the calls more readable?